### ISSUE TITLE
Marketing team page

### ISSUE THREAD

**Issue by jack-hermanson** [jack-hermanson/cis200#136]
Make sure to add a heading for the "letter" part. That should be in its own card.
<img width="1267" alt="Screen Shot 2022-01-04 at 4 45 44 PM" src="https://user-images.githubusercontent.com/51171534/148138988-d140871a-76a6-46fd-bfac-39e9073ab177.png">

**Comment by jack-hermanson** [comment]
Wait on #137
